package.json

```diff
@@ -211,7 +212,7 @@
     "yarn-deduplicate": "3.1.0"
   },
   "engines": {
-    "node": ">=18.15.0",
+    "node": ">=18.17.0",
```

Comment:
This would be non-passive to bump for yarn projects

Reply:
wait -- my bad -- this is our for our development version. This is fine. I mistook this for cli/package.json.